src/js/weather.js

```diff
@@ -1,74 +1,67 @@
 export {getLocation, ShowLocalWeather,ShowKievWeather};
+const axios = require('axios').default;
 
-let z;
-let str;
+let city;
+let cityStr;
 var deleteSymbols;
 
 function getLocation() {
     if (navigator.geolocation) {
         navigator.geolocation.getCurrentPosition(showPosition);
     } else {
         x.innerHTML = "Geolocation is not supported by this browser.";
+        
     }
 }
-function showPosition(position) {
-    let lat = position.coords.latitude;
-    let lng = position.coords.longitude;
-    var url = "https://www.mapquestapi.com/geocoding/v1/reverse?key=1rgOn51ZPNAMn2lAQvX8yQkB5rib5hKV";
-    const data = JSON.stringify({
-        location: {
-            latLng: {
-                lat: `${lat}`,
-                lng: `${lng}`
+    function showPosition(position) {
```

Comment:
Rework this local functions to one "API service" class. Methods of this class should return promises with response, no any other logic should be included.